providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/ContainerConfig.java

```diff
@@ -0,0 +1,49 @@
+package dev.openfeature.contrib.providers.flagd.e2e;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class ContainerConfig {
```

Comment:
Nice. Could you just add some comments to the `sync` and `flagd` methods explaining what they do (start the flagd/sync containers)? It's obvious to you and I, but probably not others. Also mention `unstable` starts the container which restarts itself to test reconnection.